### ISSUE TITLE
create/update kubeconfig on cluster start 

### ISSUE THREAD

**Issue by gorkem** [code-ready/crc#1216]
Can we create or update a `kubeconfig`  as part of the cluster start? We can have one cluster 2 contexts (kubeadmin & developer). If KUBECONFIG is set on the environment we can append the new files to the KUBECONFIG env. variable print its location to the crc output if the user would like to make that more permanent.  If KUBECONFIG is not set, we can set it to this new file and  $HOME/.kube/config.

**Comment by praveenkumar** [comment]
@gorkem we do have a kubeconfig file `~/.crc/machines/crc/kubeconfig` which have `system:admin` context, I think for developer and kubeadmin are the users and if we set the context on the current kubeconfig or using a new one it will based on the token which expire after ~24 hour iirc. Any specific reason we do need this?

**Comment by gorkem** [comment]
I do not think anybody keeps CRC running for longer than 24 hours :), and crc start should update the token. I was not aware that we had a kubeconfig and can't recall a reference to it on  docs. I checked the file and the context is named `admin`. It should reflect that it is crc. Perhaps `crc-admin` and `crc-developer` is more descriptive. 

There are multiple use cases that this can help with. For instance when I start CRC through tray and move to my IDE my cluster does not appear on the list until I a `oc login`. It allows me to use a tool like `kubectx` easily, especially if we can change the naming as descriptive as I explained above. It does shorten the path from `crc start` to `odo push` a couple of steps.
